Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule_test.py

```diff
@@ -619,3 +619,61 @@ def test_get_token_managed_identities__error(requests_mock, mocker):
 
     err_message = 'Error in Microsoft authorization with Azure Managed Identities'
     assert err_message in MicrosoftApiModule.return_error.call_args[0][0]
+
+
+args = {'test': 'test_arg_value'}
+params = {'test': 'test_param_value', 'test_unique': 'test_arg2_value'}
+
+
+def test_get_from_args_or_params__when_the_key_exists():
+    """
+    Given:
+        args and params with the same key in both, and a unique key in params
```

Comment:
This should be split into 2 separate tests - it is not critical but make more sense to me.

Reply:
Fixed.